xunithub.js

```diff
@@ -117,6 +118,8 @@ xunithub.prototype.markDownConverter = function (failureList) {
                 errorMD += "+ __Test Case Name :__ _" + f.name + "_\n";
                 errorMD += "+ __Failure Message :__" + "\n\n" + "   ```" + "\n"
                 + f.message + "\n" + "   ```" + "\n\n";
+                errorMD += "+ __Details :__" + "\n\n" + "   ```" + "\n"
+                + f.details + "\n" + "   ```" + "\n\n";
```

Comment:
@jj4th  should we put a condition here , if (obj.details) then only add the extra markdown ?